### DISCUSSION TITLE
qdrant vs milvus performance and stability? Which one is better?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#26735] [Q&A and General discussion] [answered]
qdrant vs milvus performance and stability? Which one is better?

I like milvus. But it not really stable especially when sudden reboot or something like that happens.
I suppose because of etcd.

**Comment by xiaofan-luan** [comment, marked as answer]
> qdrant vs milvus performance and stability? Which one is better?
> 
> I like milvus. But it not really stable especially when sudden reboot or something like that happens. I suppose because of etcd.

Thanks for the feedback~
Actually we spend a lot of time improving the stability of milvus.
It's probably worth to try Milvus 2.2.14 version as the latest stable release. 
Let us know what problem you hit(if you can offer logs ) and the community is glad to help!

Also with https://github.com/zilliztech/VectorDBBench  you can learn more about stability comparisons

**Reply by martinenkoEduard** [reply]
What is the difference between - 
Milvus-16c64g-hnsw
Milvus-2c8g-disk
Milvus-2c8g-hnsw
Milvus-4cl6g-disk


What does disk mean?

**Reply by xiaofan-luan** [reply]
disk means diskANN index.
I don't think you need to care about disk index unless you have high performance nvme machines
